content/docs/components/serving/tfserving_new.md

```diff
@@ -132,31 +124,98 @@ spec:
-   how to setup access.
+- The example contains three configurations for Google Cloud Storage (GCS) access:
+  volumes (secret `user-gcp-sa`), volumeMounts, and
+  env (GOOGLE_APPLICATION_CREDENTIALS).
+  If your model is not at GCS (e.g. using S3 from AWS), See the section below on 
+  how to setup access.
 
-1. GPU. Here the examples is using one GPU. If you don't want to use GPU, remove the
-   `nvidia.com/gpu: 1` in container resources, and use a CPU image instead of
-   `tensorflow/serving:1.11.1-gpu`.
+- GPU. If you want to use GPU, add `nvidia.com/gpu: 1`
+  in container resources, and use a GPU image, for example:
+  `tensorflow/serving:1.11.1-gpu`.
+  ```yaml
+  resources:
+    limits:
+      cpu: "4"
+      memory: 4Gi
+      nvidia.com/gpu: 1
+  ```
 
-1. The resource `VirtualService` and `DestinationRule` are for routing.
-   With the example above, the model is accessible at `HOSTNAME/tfserving/models/mnist` 
-   (HOSTNAME is your KF deployment hostname). To change the path, edit the
-   `http.match.uri` of VirtualService.
+- The resource `VirtualService` and `DestinationRule` are for routing.
+  With the example above, the model is accessible at `HOSTNAME/tfserving/models/mnist` 
+  (HOSTNAME is your Kubeflow deployment hostname). To change the path, edit the
+  `http.match.uri` of VirtualService.
 
 ### Pointing to the model
 Depending where model file is located, set correct parameters
 
 *Google cloud*
 
-Set the param as above section.
+Change the deployment spec as follows:
+
+```yaml
+spec:
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "true"
+      labels:
+        app: mnist
+        version: v1
+    spec:
+      containers:
+      - args:
+        - --port=9000
+        - --rest_api_port=8500
+        - --model_name=mnist
+        - --model_base_path=gs://kubeflow-examples-data/mnist
+        command:
+        - /usr/bin/tensorflow_model_server
+        env:
+        - name: GOOGLE_APPLICATION_CREDENTIALS
+          value: /secret/gcp-credentials/user-gcp-sa.json
+        image: tensorflow/serving:1.11.1-gpu
+        imagePullPolicy: IfNotPresent
+        livenessProbe:
+          initialDelaySeconds: 30
+          periodSeconds: 30
+          tcpSocket:
+            port: 9000
+        name: mnist
+        ports:
+        - containerPort: 9000
+        - containerPort: 8500
+        resources:
+          limits:
+            cpu: "4"
+            memory: 4Gi
+            nvidia.com/gpu: 1
+          requests:
+            cpu: "1"
+            memory: 1Gi
+        volumeMounts:
+        - mountPath: /var/config/
+          name: config-volume
+        - mountPath: /secret/gcp-credentials
+          name: gcp-credentials
+      volumes:
+      - configMap:
+          name: mnist-v1-config
+        name: config-volume
+      - name: gcp-credentials
+        secret:
+          secretName: user-gcp-sa
+```
+
+The changes are:
```

Comment:
Add an empty line before the list, to fix the rendering problem:
https://deploy-preview-1025--competent-brattain-de2d6d.netlify.com/docs/components/serving/tfserving_new/

Reply:
fixed, thanks